heartbeat/docs/fields.asciidoc

```diff
@@ -7967,6 +7967,17 @@ example: 2019-08-16 01:40:25+00:00
 
 --
 
+*`tls.server.x509.days_to_expiry`*::
```

Comment:
Any reason to prefer days to millis? That should easily fit in a long and will probably be more useful.

Reply:
Done